content/rs/databases/active-active/manage-aa.md

```diff
@@ -0,0 +1,77 @@
+---
+Title: Manage Active-Active databases 
+description: Manage Active-Active
+weight: 30
+alwaysopen: false
+categories: ["RS"]
+aliases: [
+    /rs/databases/active-active/edit-aa-database.md,
+    /rs/databases/active-active/edit-aa-database/,
+    /rs/databases/active-active/manage-aa.md,
+    /rs/databases/active-active/manage-aa/,
+]
+---
+
+You can configure and manage your Active-Active database from either the admin console or the command line
+
+When you edit the database configurations of an Active-Active database with the admin console or the `rladmin` CLI, the changes will only apply to the Active-Active database instance you are currently editing.
+
+To change the global configuration of the Active-Active database, use the [`crdb-cli`]({{<relref "/rs/references/cli-utilities/crdb-cli">}}).
+
+## Database settings
+
+Following table shows a list of database settings, tools you can use to change those settings, and links to more information.
```

Comment:
```suggestion
The following table shows a list of database settings, tools you can use to change those settings, and links to more information.
```

Also, I don't see a table on this page - either it's missing, or this sentence should be removed